src/platform_impl/web/web_sys/canvas/pointer_handler.rs

```diff
@@ -103,11 +111,91 @@ impl PointerHandler {
         ));
     }
 
+    pub fn on_pointer_move<F>(&mut self, canvas_common: &super::Common, mut handler: F)
+    where
+        F: 'static + FnMut(i32, PhysicalPosition<f64>),
+    {
+        self.on_pointer_move = Some(canvas_common.add_event(
+            "pointermove",
+            move |event: PointerEvent| {
+                handler(
+                    event.pointer_id(),
```

Comment:
The handler must be called only when `event.pointer_type()` is equal to `PointerType::Touch`

Reply:
Changed!